src/components/Header/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import { CONSTANTS } from "../../constants";
+import { SearchFilm } from "../SearchFilm";
+import "./style.scss";
+
+interface HeaderProps {
+  onSearch(event: React.ChangeEvent<HTMLFormElement>): void;
+  openModal(type: string, filmID: string): void;
+}
+
+const Header: React.FC<HeaderProps> = ({ onSearch, openModal }) => {
+  return (
+    <div className="header">
+      <div className="header-nav">
+        <h3>
+          <span>netflix</span>roulette
+        </h3>
+        <button onClick={openModal.bind(null, CONSTANTS.FORM_TYPE.ADD, "")}>
```

Comment:
so what the reason of this structure?
sometimes there are cases where you can use arrow functions, like this, when you have dynamic params. But still you can create `useCallback` here with specific params, and avoid arrow funct
```
const openAddMovieModal = useCallback(()=>{
   openModal(CONSTANTS.FORM_TYPE.ADD, "");
}, []);
```